release-content/migration_guides_template.md

```diff
@@ -30,10 +30,22 @@ Keep it short and sweet:
 - Use bullet points to explain complex changes.
 - Avoid headings. If you must, use only level-two headings.
 - Diff codeblocks can be useful for succinctly communicating changes.
-
-```diff
-fn my_system(world: &mut World) {
-+ world.new_method();
-- world.old_method();
-}
-```
+  
+  ```diff
+   fn my_system(world: &mut World) {
+  +    world.new_method();
+  -    world.old_method();
+   }
```

Comment:
[I could have](https://github.com/bevyengine/bevy/pull/18654/files#r2022231635) added the suggestion instead 😅 
```suggestion
  fn my_system(world: &mut World) {
  +    world.new_method();
  -    world.old_method();
  }
```

Reply:
Is there any reason we place addition before deletion?

Reply:
I don't think it matters. 🤷‍♂️